src/encoded/static/components/item-pages/components/tables/ItemPageTable.js

```diff
@@ -75,10 +77,12 @@ export class EmbeddedItemSearchTable extends React.PureComponent {
             termTransformFxn: Term.toName
         };
 
+        const externalLinkVisible = (typeof externalSearchLinkVisible === 'boolean' && externalSearchLinkVisible) || false;
```

Comment:
FYI: The `searchHref` we get here is the same exact value that we originally pass into it, not the current searchHref based on current filters active (if any).

We've [very recently updated this](https://github.com/dbmi-bgm/cgap-portal/blob/9a3b7ef3178136b004b9f63a5cd203c4c2c524ae/src/encoded/static/components/item-pages/components/EmbeddedItemSearchTable.js#L78-L90) `EmbeddedItemSearchTable` component on CGAP-side (as well as the SPC component it renders) to instead of rendering showTitle, to pass down `props.children` or (_maybe_ to-be-deprecated) `props.title` into the `EmbeddedSearchView` as `props.embeddedTableHeader` where the JSX component that's represented by `props.embeddedTableHeader` will get cloned with the _real_ / _current_ props.searchHref (of current filters) being passed into it.

At this point, it's probably better to copy/paste/migrate [those changes from CGAP](https://github.com/dbmi-bgm/cgap-portal/commit/654d5245fd2f73430503c8128229fb1e00b8b070#diff-b258ddd7ee2ce2fcbfbeadfc793c68ba) into 4DN and then the current `"title":  <ExperimentSetsTableTabViewTitle/>` will just work with most up-to-date searchHref and no need for `externalLinkVisible ` (can define externalLinkVisible, if is still needed, simply in `<ExperimentSetsTableTabViewTitle externalLinkVisible={true} />` as needed.)

Reply:
@alexkb0009 thanks for the feedback!

1. Re: FYI: The searchHref we get here is the same exact value that we originally pass into it, not the current searchHref based on current filters active (if any).

I think, the original url we passed,`searchHref`,  is ok in this case since we aim to navigate the user to display the initial results when `Open In Browse/Search` View button is clicked.

2. Great! We'll migrate the changes you did in CGAP into 4DN shortly. (externalLinkVisible will remain)